src/Webhooks/Events/OnGlobalChatMessage.cs

```diff
@@ -8,11 +9,24 @@ public class OnGlobalChatMessage : IScript
         [Target(GameSourceEvent.PlayerGlobalChatMessage, ExecutionMode.Event)]
         public void OnEvent(ShPlayer player, string message)
         {
+            List<Embed> commandsEmb = new List<Embed>();
+            List<Embed> globalEmb = new List<Embed>();
```

Comment:
Use var where ever possible 
```suggestion
           var commandsEmb = new List<Embed>();
           var globalEmb = new List<Embed>();
```